backend/api/Models/Parcel/ParcelBuildingModel.cs

```diff
@@ -31,6 +32,16 @@ public class ParcelBuildingModel : BaseModel
 
         public string BuildingPredominateUse { get; set; }
 
+        public int BuildingOccupantTypeId { get; set; }
+
+        public string BuildingOccupantType { get; set; }
+
+        public DateTime? LeaseExpiry { get; set; }
+
+        public string OccupantName { get; set; }
+
+        public bool? TransferLeaseOnSale { get; set; }
```

Comment:
I noticed the other models didn't make this nullable.

Reply:
they should.